lagtraj/input_definitions/load.py

```diff
@@ -82,6 +80,11 @@ def load_definition(input_name, input_type, root_data_path, required_fields):
                     " a directory structure as follows (so that"
                     " so that the relevant input data and output"
                     f" can be found by lagtraj): \n{FOLDER_STRUCTURE_EXAMPLE}"
+                    "Files under input_examples can be retrieved as well, "
```

Comment:
I would change this to use `lagtraj.input_definitions.examples.get_available()` instead so that we don't mention the folder `input_examples/` by name. So something like

```python
from lagtraj.input_definitions.examples import get_available as get_available_input_examples

...

lagtraj_input_examples = get_available_input_examples(input_types=input_type)
s = ", ".join(lagtraj_input_examples[:3])  #  show the first three only

raise Exception(
    ...
    f"{input_type_plural} input definitions bundled with `lagtraj` can be used as well (for example {s}"
    " - see all available with `python -m lagtraj.input_definitions.examples`)
    "e.g. \n"
    "python -m lagtraj.trajectory.create lagtraj://eurec4a_20200202_12_lag \n"
    "The input files are then copied over to the data directory."
)